### DISCUSSION TITLE
Cloudfared ne marche pas clifty

### DISCUSSION THREAD

**Discussion by Karim99z** [Alygnt/Clifty#89] [Q&A]
Hello when I select cloudfared on clifty it tells me "error generating link "help please

**Comment by Karim99z** [comment]
all the other links also give me this error

**Reply by Alygnt** [reply]
Working on it... Meanwhile, NGROK is fixed and is completely stable now. You can use that until cloudflared will be fixed.

**Comment by Karim99z** [comment]
Apars the local
